### DISCUSSION TITLE
Loading Zarr dataset ?

### DISCUSSION THREAD

**Discussion by steph-ben** [developmentseed/titiler#461] [Q&A] [answered]
Hi there,

I'm looking for rendering zarr dataset through titiler.
Zarr data can be open easily with xarray.open_dataset()

I'm not sure : is this possible with Titiler ?

Many thanks for your great work !
Stéphane

**Comment by jsuwala** [comment, marked as answer]
This looks to be "coming soon". See https://github.com/cogeotiff/rio-tiler/pull/530
